.buildkite/pipeline.yml

```diff
@@ -40,18 +42,17 @@ steps:
       distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
       hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
 
+  # Since make test-images mutates the source directory, it cannot be run with other steps.
+  - wait
+
   - label: ':linux: build docker images'
     commands:
-      - 'make test-images'
-      - 'buildkite-agent artifact upload testdata/firecracker-master'
-      - 'buildkite-agent artifact upload testdata/jailer-master'
```

Comment:
out of curiosity, why are we not `artifact upload`-ing these anymore?

Reply:
We were using that to pass artifacts between tests, but now we use `/tmp/buildkite_build_${BUILDKITE_BUILD_NUMBER}_testdata` instead.